build.zig

```diff
@@ -21,10 +21,10 @@ pub fn build(b: *std.Build) !void {
 	exe.linkLibC();
 	exe.linkLibCpp();
 
-	const depsLib = b.fmt("cubyz_deps_{s}-{s}-{s}", .{@tagName(t.cpu.arch), @tagName(t.os.tag), switch(t.os.tag) {
-		.linux => "musl",
-		.macos => "none",
-		.windows => "gnu",
+	const depsLib = b.fmt("libcubyz_deps_{s}-{s}-{s}", .{@tagName(t.cpu.arch), @tagName(t.os.tag), switch(t.os.tag) {
+		.linux => "musl.a",
+		.macos => "none.a",
+		.windows => "gnu.a",
```

Comment:
This doesn't work on windows, since the name and extension of the static lib are different.

Reply:
I'll add it in another case.